x/wasm/ibc.go

```diff
@@ -18,6 +20,11 @@ import (
 	"github.com/CosmWasm/wasmd/x/wasm/types"
 )
 
+// DefaultMaxIBCCallbackGas is the default value of maximum gas that an IBC callback can use.
+// If the callback uses more gas, it will be out of gas and will not be reverted.
```

Comment:
"will not be reverted" is this correct? Sounds scary if some half done callback is not reverted

Reply:
Ok, the sentence is not good. What I mean is: The transaction is not reverted. The contract state is reverted though.
It basically refers to this paragraph of the [ADR-8 spec](https://github.com/cosmos/ibc-go/blob/8c79fac7e69b67ed697bc299657cd26ab2439407/docs/architecture/adr-008-app-caller-cbs.md?plain=1#L255-L256): 
> If the relayer-defined gas limit is exceeded before the user-defined gas limit or global callback gas limit is exceeded, then the entire transaction is reverted to allow for resubmission. If the chain-defined or user-defined gas limit is reached, 
the callback state changes are reverted and the transaction is committed.

I'll change the wording to explain that better.